### DISCUSSION TITLE
android tv

### DISCUSSION THREAD

**Discussion by crowley88** [pear-devs/pear-desktop#996] [General]
Would there be any way to port this to android tv? It would be appreciated by many as there is a serious gap in the market for an application like this. Android TV has smarttube for normal YouTube but seriously lacks for YouTube music and this is the ultimate YouTube music alternative

**Comment by Zo-Bro-23** [comment]
+1 for this! 

**_But_**, we'll just have to do the entire app again. There's no way AFAIK to run Electron apps on Android or Android TV, so we can't really _port_ the app. We'll just have to make a new app for Android, which is way beyond the scope of this project.

There might be a way to run Electron on Android though, especially in the future ([this](https://stackoverflow.com/questions/71415088/is-it-possible-to-deploy-an-app-you-built-with-electron-to-android) looks promising). Feel free to look into it if you're interested!
